internal/apps/dop/providers/issue/dao/issue_relation.go

```diff
@@ -177,21 +177,21 @@ type Counter struct {
 }
 
 // GetRequirementInclusionTaskNum get the number of requirements associated with the task in this iteration
-func (client *DBClient) GetRequirementInclusionTaskNum(iterationID uint64) (uint64, error) {
-	var count Counter
-	if err := client.Table("dice_issue_relation").Select("count(distinct(issue_id)) as count").Where(fmt.Sprintf("issue_id in (select id from dice_issues where iteration_id='%d' and type='REQUIREMENT' and deleted=0) and type='inclusion'", iterationID)).
-		Find(&count).Error; err != nil {
-		return 0, err
+func (client *DBClient) GetRequirementInclusionTaskNum(iterationID uint64) (uint64, []uint64, error) {
```

Comment:
It's better to use named vars in function signature directly.

Reply:
ok, you're right